lambdas/initialise-ipv-session/src/main/java/uk/gov/di/ipv/core/initialiseipvsession/validation/JarValidator.java

```diff
@@ -169,11 +173,13 @@ private JWTClaimsSet getValidatedClaimSet(SignedJWT signedJWT, String clientId)
                                 JWTClaimNames.SUBJECT));
 
         try {
-            verifier.verify(signedJWT.getJWTClaimsSet(), null);
+            JWTClaimsSet claimsSet = signedJWT.getJWTClaimsSet();
+            verifier.verify(claimsSet, null);
 
-            validateMaxAllowedJarTtl(signedJWT.getJWTClaimsSet());
+            validateMaxAllowedJarTtl(claimsSet);
+            validateInheritedIdentityJwtClaim(claimsSet);
```

Comment:
Can we put this behind the feature flag, to avoid the risk of breaking any prod journeys prematurely?

Reply:
Have moved validation to only be in `validateAndStoreHMRCInheritedIdentity`